bokeh/validation/errors.py

```diff
@@ -0,0 +1,18 @@
+''' Define standard error codes and messages for Bokeh validation checks.
+
+'''
+
+codes = {
+    1000: ("COLUMN_LENGTHS",        "ColumnDataSource column lengths are not all the same"),
+    1001: ("BAD_COLUMN_NAME",       "Glyph refers to nonexistent column name"),
+    1002: ("MISSING_GLYPH",         "Glyph renderer has no glyph set"),
+    1003: ("NO_SOURCE_FOR_GLYPH",   "Glyph renderer has no data source"),
+    9999: ("EXT",                   "Custom extension reports error"),
+}
+
+for code in codes:
+    exec("%s = %d" % (codes[code][0], code))
```

Comment:
I don't really have a strong feeling either way. I'm personally ok with `exec` as long as the exec'ed contents are controlled and user input is nowhere near it.